.travis.yml

```diff
@@ -35,9 +38,12 @@ before_script:
 script:
   - bash tests/bin/phpunit.sh
   - bash tests/bin/phpcs.sh
+  - bash tests/e2e-tests/scripts/run-e2e.sh  
 
 after_script:
   - bash tests/bin/travis.sh after
+  - export NODE_CONFIG_DIR="./tests/e2e-tests/config"
+  - ./tests/e2e-tests/scripts/wp-serverpilot-delete.js
```

Comment:
Should we maybe add a check to call this script only when `RUN_E2E` is true? This way we would avoid calling it on jobs where we don't run e2e tests. This is producing the error below:

https://travis-ci.org/woocommerce/woocommerce/jobs/326418964#L303